addons/website/static/src/core/colibri.js

```diff
@@ -185,7 +185,11 @@ export class Colibri {
             } else if (directive === "t-out") {
                 this.tOuts.push([nodes, value]);
             } else if (directive === "t-component") {
-                this.mountComponent(nodes, value);
+                if (!!value.prototype) {
+                    this.mountComponent(nodes, value);
+                } else {
+                    this.mountComponent(nodes, ...value());
+                }
```

Comment:
Don't know about that. Maybe we can just keep the `() => ([Component, props])` syntax